frontend/lib/verification/verification_card_details.dart

```diff
@@ -0,0 +1,8 @@
+import '../identification/base_card_details.dart';
+
+class VerificationCardDetails {
+  final BaseCardDetails cardDetails;
+  final int otp;
+
+  VerificationCardDetails(this.cardDetails, this.otp);
```

Comment:
Why not inheritance?

Reply:
In this example I already have a CardDetails instance: https://github.com/ehrenamtskarte/ehrenamtskarte/blob/56cffe71fc9685de1d0f92aa99f3625ea7b6aff5/frontend/lib/identification/card_detail_view.dart#L112

From what I understand I would need to call the super constructor from basecarddetails which means that I need to pass every single parameter again, effectively making a copy. Is there a way to create a sub class from an instance of a super class? Never though about that :D